example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -388,9 +408,19 @@ describe('The S3 Ingest Granules workflow', () => {
 
   describe('the BackupGranulesToLzards task', () => {
     let lambdaOutput;
-
+    let subTestSetupError;
     beforeAll(async () => {
-      lambdaOutput = await lambdaStep.getStepOutput(workflowExecutionArn, 'LzardsBackup');
+      try {
+        if (beforeAllError || subTestSetupError) throw SetupError;
+        lambdaOutput = await lambdaStep.getStepOutput(workflowExecutionArn, 'LzardsBackup');
+      } catch (error) {
+        subTestSetupError = error;
+      }
+    });
+
+    beforeEach(() => {
```

Comment:
Do we need a beforeEach if there is only one test in this describe block? Or is it for future proofing?

Reply:
We don't, it was mostly for consistency/ease later given I was doing a ton of jasmine-esque boilerplate.